client/components/ErrandTracker/BottomSheet/ErrandProgress.jsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { ProgressBar } from 'react-native-paper';
+import { StyleSheet } from 'react-native';
+
+const ErrandProgress = () => (
+  <ErrandProgress progress={0.5} color="#00BCD4" style={styles.bar} />
```

Comment:
Would this be <ProgressBar /> instead of <ErrandProgress />? I don't know how ProgressBar works though so I could definitely be off base

Reply:
I actually am planning on using another dependency for progress bar and haven't completely done the conditionals yet lol. Was planning on commenting out the Component for now